code/game/machinery/doors/door.dm

```diff
@@ -93,6 +94,23 @@
 	air_update_turf(TRUE, FALSE)
 	return ..()
 
+/**
+ * Signal handler for checking if we notify our surrounding that access requirements are lifted accordingly to a newly set security level
+ *
+ * Arguments:
+ * * source The datum source of the signal
+ * * new_level The new security level that is in effect
+ */
+/obj/machinery/door/proc/check_security_level(datum/source, new_level)
+	SIGNAL_HANDLER
+
+	if(new_level <= SEC_LEVEL_BLUE)
+		return
+	if(!red_alert_access)
+		return
+	visible_message("<span class='notice'>[src] whirrs as it automatically lifts access requirements!</span>")
```

Comment:
```suggestion
	visible_message("<span class='notice'>[src] whirrs as [p_they()] automatically lifts access requirements!</span>")
```

Reply:
Think you'll need to add an `[p_s()]` to the s on lifts too?

Reply:
i know this is for double doors but i can't help but imagine "bob the Man Door, half man half door"

Reply:
Added `[p_s()]` to both.

Reply:
I had no idea `p_s` was a thing. 😛